Firebase/Core/FIRNetworkURLSession.m

```diff
@@ -237,13 +237,15 @@ - (void)URLSession:(NSURLSession *)session
   }
 }
 
+#if TARGET_OS_IOS || TARGET_OS_TV
```

Comment:
What will happen on macOS?

Reply:
It seems that macOS doesn't have a way to say that all messages enqueued for a session have been delivered. The method will never be called.

See https://developer.apple.com/documentation/foundation/nsurlsessiondelegate?language=objc